ui/src/components/data-entities/form-reducer.js

```diff
@@ -33,6 +34,21 @@ const formSlice = createSlice({
     itemLoaded: (state, action) => {
       state.editItem = action.payload;
     },
+    setSelectedFormData: (state, action) => {
+      let resp = {};
+      const key = Object.keys(action.payload)[0];
+      resp[key + "Selected"] = action.payload;
+      resp[key] = action.payload[key]._links.self.href;
+      state.editItem = {...state.editItem, ...resp};
+    },
+    selectedItemsLoaded: (state, action) => {
+      let resp = {};
+      const key = Object.keys(action.payload._embedded)[0];
+      resp[key] = action.payload._embedded;
+      resp[pluralize.singular(key) + "Selected"] = action.payload.selected;
+      resp[pluralize.singular(key)] = action.payload.selected._links.self.href;
+      state.editItem = {...state.editItem, ...resp};
+    },
```

Comment:
I have pull the branch locally and it's crashing here. I don't see where action.payload.selected get fill ?

Especially in the case of new element.